gradle.properties

```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=1.6.7-SNAPSHOT
+VERSION_NAME=1.6.8-SNAPSHOT
```

Comment:
You might consider changing this since the latest version available is **1.6.64**